tests/foreman/ui/test_host.py

```diff
@@ -2311,28 +2311,53 @@ def test_positive_gce_cloudinit_provision_end_to_end(
             googleclient.disconnect()
 
 
-@pytest.mark.destructive
 @pytest.mark.upgrade
 @pytest.mark.usefixtures('install_cockpit_plugin')
 @pytest.mark.tier2
-def test_positive_cockpit(session, default_sat):
-    """Install cockpit plugin and test whether webconsole button and cockpit integration works
+def test_positive_cockpit(destructive_sat):
+    """Install cockpit plugin and test whether webconsole button and cockpit integration works.
+    also verify if cockpit service is restarted after the service restart.
 
     :id: 5a9be063-cdc4-43ce-91b9-7608fbebf8bb
 
-    :expectedresults: Cockpit page is loaded and displays sat host info
+    :expectedresults: 1.cockpit service is restarted after the services restart.
```

Comment:
As per the process we follow in Robottelo tests, the `expectedresults` should always be placed after `steps`, please update the same here for this test case!

Reply:
Will update it